Yosemite/Yosemite/Actions/ShippingLabelAction.swift

```diff
@@ -11,4 +11,11 @@ public enum ShippingLabelAction: Action {
                             shippingLabelID: Int64,
                             paperSize: ShippingLabelPaperSize,
                             completion: (Result<ShippingLabelPrintData, Error>) -> Void)
+
+    /// Requests a refund for a shipping label.
+    ///
+    case refundShippingLabel(siteID: Int64,
+                             orderID: Int64,
+                             shippingLabelID: Int64,
```

Comment:
I wonder if we should also consider submitting the `ShippingLabel` object. As I was testing this manually, I ended up just using the object anyway. 🤷 

```swift
            guard let shippingLabel = dataSource.shippingLabel(at: indexPath) else {
                return
            }

            // This could just pass the `ShippingLabel`
            let action = ShippingLabelAction.refundShippingLabel(siteID: shippingLabel.siteID,
                                                                 orderID: shippingLabel.orderID,
                                                                 shippingLabelID: shippingLabel.shippingLabelID) { result in
                print(result)
                print("done!")
            }
            print("refunding \(shippingLabel)")
            ServiceLocator.stores.dispatch(action)
```

Probably something to consider once everything is connected.

Reply:
Makes sense, it does save a bunch of code! Updated in: https://github.com/woocommerce/woocommerce-ios/pull/3294/commits/1686982f6e315cf92c3fca7c0d1c58671dc6dde2